docs/source/contrib_guide_bugs_enh.rst

```diff
@@ -0,0 +1,57 @@
+=================
+Submitting a Bug
+=================
+While using the Notebook, you might experience a bug that manifests itself in 
+unexpected behavior.  If so, we encourage you  to open issues on Github. To 
+make the navigating issues eaasier for both developers and users, we ask that
+you take the following steps before submitting an issue.
+
+1. Search through StackOverflow and existing Github issues to ensure that 
+the issue has not already been reported by another user. If so, provide
+your input on the existing issue if you think it would be valuable.
+
+2. Prepare a small, self-contained snippet of code that will allow others
+to reproduce the issue that you are experiencing.
+
+3. Prepare information about the environment that you are executing the code
+in, in order to aid in the debugging of the issue. You can use ``python 
+-c "import IPython; print(IPython.sys_info())"`` to get some initial
```

Comment:
sys_info() is still in IPython, though, it will only give version of IPython, not notebook, jupyter, jupyter_client, or other. 
We would have to build a tool that collect such info.